lms/models/lti_user.py

```diff
@@ -1,11 +1,20 @@
 from dataclasses import dataclass
-from typing import List
+from typing import List, Optional
 
 from lms.models.application_instance import ApplicationInstance
 from lms.models.h_user import HUser
 from lms.models.lti_role import LTIRole, RoleScope, RoleType
 
 
+@dataclass
+class LTI:
+    course_id: str
+    """ID of the course in the LMS, context_id using LTI naming"""
+
+    assignment_id: Optional[str] = None
```

Comment:
I haven't created `Course` and `Assignment` dataclasses just for the ID.

It should be possible to morph them into that when needed, maybe using the same serialization as now.